tests/common/lib.bash

```diff
@@ -129,6 +129,20 @@ yq_dig() {
   fi
 }
 
+# usage: yq_secret <config-key> <default>
+yq_secret() {
+  if [[ -n "${1:-}" ]]; then
```

Comment:
suggestion:

```suggestion
  if [[ 0 == $# ]]; then
    fail "missing config key argument"
    # assuming this also returns from the function?
```

inverting the condition allows an early return and dropping one level of indentation